client/blocks/upe/styles.scss

```
@@ -39,6 +39,39 @@ button.stripe-gateway-stripelink-modal-trigger:hover {
 
+/* Typography/sizing matches Stripe's Payment Element redirect notice so ACSS
+ * looks consistent with other redirect-based methods (e.g. Afterpay).
+ */
+.wc-stripe-acss-notice {
+	display: flex;
+	align-items: center;
+	gap: 12px;
+	margin: 0;
+	padding: 24px 9px 8px;
+	color: rgb(43, 45, 47);
+	font-family: 'Source Sans Pro', 'HelveticaNeue-Light', 'Helvetica Neue Light', 'Helvetica Neue', 'Helvetica', 'Arial', 'Lucida Grande', sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "HelveticaNeue-Light" (font-family-name-quotes)

(font-family-name-quotes)